AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -245,7 +245,7 @@ - (void)dealloc
 
 - (UIResponder *)nextResponder
 {
-  return self.view.superview;
+  return [self.view nextResponder];
```

Comment:
Ran through unit tests and Kittens. Let me spin up the other sample projects to be extra sure.